tests/integration/test_examples.py

```diff
@@ -19,20 +20,28 @@
 from zenml.cli import EXAMPLES_RUN_SCRIPT, LocalExample
 from zenml.core.repo import Repository
 from zenml.enums import ExecutionStatus
+from zenml.logger import get_logger
+
+logger = get_logger(__name__)
+
 
 QUICKSTART = "quickstart"
 NOT_SO_QUICKSTART = "not_so_quickstart"
 CACHING = "caching"
 
 
-@pytest.fixture(scope="session")
+@pytest.fixture(scope="module")
 def examples_dir(tmp_path_factory):
+    prev_wd = os.getcwd()
```

Comment:
This might be a conflict with my branch so we should resolve it when that happens

Reply:
Definitely no longer necessary with your branch, will remove it.